src/simplewallet/simplewallet.cpp

```diff
@@ -2974,6 +2974,22 @@ bool simple_wallet::init(const boost::program_options::variables_map& vm)
       // create wallet
       bool r = new_wallet(vm, "Ledger");
       CHECK_AND_ASSERT_MES(r, false, tr("account creation failed"));
+      // if no block_height is specified, assume its a new account and start it "now"
+      if(m_wallet->get_refresh_from_block_height() == 0) {
+        {
+          tools::scoped_message_writer wrt = tools::msg_writer();
```

Comment:
Get a reference, just to be safe ?

Reply:
Just only one contruction for multi line.

Reply:
Why not write this way `tools::msg_writer() << ...` like the rest of the code?

Reply:
ok